src/SprykerEco/Zed/Computop/Communication/Controller/GatewayController.php

```diff
@@ -24,7 +24,7 @@ class GatewayController extends AbstractGatewayController
      */
     public function logResponseAction(ComputopApiResponseHeaderTransfer $headerTransfer): ComputopApiResponseHeaderTransfer
     {
-        return $this->getFacade()->logResponseHeader($headerTransfer, $headerTransfer->getMethod());
+        return $this->getFacade()->logResponseHeader($headerTransfer, $headerTransfer->getMethodOrFail());
```

Comment:
this change is bc-breaking
`$method` does not have typehint so null is technically possible
please revert this change

Reply:
`logResponseHeader()` docblock has `@param string $method` defined. So we want to allow null 
despite this?

Reply:
Fixed